src/ComponentCover/index.js

```diff
@@ -17,7 +19,7 @@ const ComponentCover = ({ children, className }) => (
                 height: inherit;
                 width: inherit;
 
-                z-index: 900;
+                z-index: ${layers.applicationTop - 1};
```

Comment:
The `ComponentCover` belongs between the application base layer and the application top layer.

We want it to cover all the components in the application base layer, but allow dropdown menus to render above it. This seems like an OK workaround.